server/web/home/index.js

```diff
@@ -22,7 +22,7 @@ exports.register = function (server, options, next) {
         authorized = true;
       }
 
-      reply.view('home/index', _assign(defaults.testPageContext, {
+      reply.view('home/index', Hoek.applyToDefaults(defaults.testPageContext, {
```

Comment:
Or, we could stick to `_assign` but use `_assign({}, defaults.testPageContext, {…})`